### ISSUE TITLE
Chinese verison of paper

### ISSUE THREAD

**Issue by yamiefun** [weixk2015/DeepSFM#9]
Hi,
Have you published the paper in Chinese?

Thanks very much.

**Comment by weixk2015** [comment]
Hi, sorry but no Chinese version is published yet and actually we don't have any plan to publish a Chinese version now.

**Comment by yamiefun** [comment]
I have some question about the cost volume described in the paper.
In the supplemental materials, in the cost volume P-CV paragraph, you said "And the initial reference view depth map is projected to align numeric values with the warped source view depth map." But in the next paragraph about D-CV, you said "And the depth plane is projected to align with the source view depth map".
 I wonder which one is correct? Align with "warped source view depth map" or "source view depth map"?

**Comment by weixk2015** [comment]
Sorry for the ambiguity in the paper. The initial reference view depth map/The depth plane is projected to align numeric values with the warped source view depth map. It is a typo in the D-CV paragraph, which should be "the depth plane is projected to align with the warped source view depth map"

**Comment by yamiefun** [comment]
Thanks very much! 
In the Architecture paragraph talking about depth consistency, you said that "The second channels contains the depth values of the virtual planes in the target view by seeing them from the source view". I don't understand this. The depth value of a specific virtual plane should be a constant right? How do I make a constant number to a W*H size slice and concatenate it to the cost volume? Do I misunderstand anything?

**Comment by weixk2015** [comment]
Yes, a plane is a tensor of size W*H and all pixels are set as the same virtual depth value in the camera coordinate system of the target view. The plane(z coordinate) is transformed to the camera coordinate system of the source view following the camera projection equation.

**Comment by yamiefun** [comment]
Thanks! I got it.

**Comment by yamiefun** [comment]
Hi, I have another question about the loss function in paragraph 3.5. I don't know what is "predicted coarse depth map and refined depth map". I don't understand where can you get the coarse depth map in your architecture? In paragraph 3.4, you will get depth maps after cost aggregation and soft-argmax, which is correspond to the "Refinement" part in Fig2, right?

Or the coarse depth map means the initial depth map, which is provided by DeMoN dataset?

**Comment by weixk2015** [comment]
The coarse depth map in 3.5 refers to the depth maps after cost aggregation and soft-argmax
